libs/payments/ui/src/lib/actions/index.ts

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Use this file to export React client components (e.g. those with 'use client' directive) or other non-server utilities
+
+export * from './checkoutCartWithPaypal';
```

Comment:
All actions are currently already exported from `ui/src/server.ts`, and should only be exported from one barrel file.

I'd recommend we continue using `ui/src/server.ts`, however if you prefer, we could start using this file, but then should update all references. (This could also be done as a quick follow up ticket)

Reply:
the current export structure of the server.ts barrel file includes server-only exports. Since importing from a barrel file also sneakily imports everything else within that barrel file, this caused build errors within nextjs. The workable changes I see are: import from the file directly without a barrel file (ugly import routes, but no unexpected dependencies), move the server actions out of the `ui/src/server.ts` barrel file, or remove the functionality from this component.

Thoughts?

Reply:
Server actions should only be executed on the server, and shouldn't be bundled up with client code, so that's why we try to have separate barrel files for client and server exports where it's necessary.

When you get an error saying something has server-only exports, that's usually a good indication that it shouldn't be used in a client component.

I probably should have mentioned this in my previous comments, but to help ensure the server / client barrier isn't accidentally crossed, we have server action that calls a method in `NextJSActionsService`, to help us as developers ensure we don't accidentally include what's meant to be a server action, in a client component.

Reply:
Lets discuss this more on Slack or in a call

Reply:
Ah sweet, this was a misunderstanding on my part. Thanks for the chat @david1alvarez

Reply:
following off-github conversation: leaving as-is, having an `actions` barrel file currently provides necessary functionality